capstone/src/main/java/com/google/sps/servlets/UpdateAssociationServlet.java

```diff
@@ -12,23 +18,56 @@
 @WebServlet("/update-associations")
 public class UpdateAssociationServlet extends HttpServlet {
 
+  public static final String SURVEY_ENTITY_KIND = "Actual";
+  public static final String COMMENT_PROPERTY = "text";
+  public static final String RESULT_ENTITY_KIND = "AssociationResult";
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
May want to initiate the datastore service in the servlet's init method, so that the service does not start every time the url is hit

Reply:
Moved the datastore and cloudnlp clients to init!